components/script/dom/file.rs

```diff
@@ -73,7 +73,8 @@ impl File {
         let ref typeString = blobPropertyBag.type_;
 
         let modified = filePropertyBag.lastModified;
-        Ok(File::new(global, BlobImpl::new_from_bytes(bytes), filename, modified, typeString))
+        let replaced_filename = DOMString::from_string(filename.replace("/", ":"));
```

Comment:
Oh sorry, I was talking about https://github.com/w3c/FileAPI/issues/41 (the bug that wishes to remove this behavior)